src/components/entropy/index.js

```diff
@@ -290,7 +290,9 @@ class Entropy extends React.Component {
           width={this.props.width}
           height={this.props.height}
         >
-          <g ref={(c) => { this.d3entropy = c; }} id="d3entropy"/>
+          <g><g><g><g>
```

Comment:
I wonder if these guards warrant an explanatory comment with the issue number …if nothing else, it will make it easier to excise them down the road, should we end up needing to.

Reply:
Added comments:

https://github.com/nextstrain/auspice/blob/17b3f5c08af54c618de64d8673bd6adb5b16abcb/src/components/entropy/index.js#L293-L295